packages/fast-components-styles-msft/src/breadcrumb/index.ts

```diff
@@ -0,0 +1,54 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { applyLocalizedProperty, Direction } from "@microsoft/fast-jss-utilities";
+import { ensureBrandNormal, ensureForegroundNormal } from "../utilities/colors";
+import { BreadcrumbClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import { applyTypeRampConfig } from "../utilities/typography";
+import { fontWeight } from "../utilities/fonts";
+
+const styles: ComponentStyles<BreadcrumbClassNameContract, DesignSystem> = (
+    config: DesignSystem
+): ComponentStyleSheet<BreadcrumbClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
+
+    return {
+        breadcrumb: {
+            padding: "4px",

```

Comment:
As a principle - we typically don't include padding or margin on the root of components; that way, we don't have to override existing values every time someone wants to use those values to position something contextually.